meta-ibm/recipes-phosphor/ipmi/phosphor-ipmi-config/p10bmc/changeIana.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+model=($(busctl get-property xyz.openbmc_project.Inventory.Manager /xyz/openbmc_project/inventory/system xyz.openbmc_project.Inventory.Decorator.Asset SubModel))
+DIR=/usr/share/ipmi-providers
+if [ "${model[1]}" = '"S0"' ]  || [ "${model[1]}" = '"D0"' ]; then
+    echo "Using IBM IANA"
+elif [ "${model[1]}" = '"J0"' ]; then
+    echo "loading IPS IANA"
+    mount --bind ${DIR}/dev_id_ips.json ${DIR}/dev_id.json
+else
+    echo "loading OEM IANA"
+fi
```

Comment:
Do we get different IANA numbers if 'model != "J0"'?

All I can see is echo different strings when model is S0/D0 and in the else part...

If not, we can just check something like:
if [ "${model[1]}" = '"J0"' ]; then
    echo "loading IPS IANA"
    mount --bind ${DIR}/dev_id_ips.json ${DIR}/dev_id.json
fi

Reply:
> Do we get different IANA numbers if 'model != "J0"'?

I think Rainier will support OEM Systems as well, like this:
https://github.com/ibm-openbmc/openbmc/blob/1020.20/meta-ibm/recipes-phosphor/webui/webui-vue/changeLogo.sh

> 
> All I can see is echo different strings when model is S0/D0 and in the else part...
> 
> If not, we can just check something like: if [ "${model[1]}" = '"J0"' ]; then echo "loading IPS IANA" mount --bind DIR/devidips.json{DIR}/dev_id.json fi

@geissonator @mzipse 
What do you think?

Reply:
I believe our new plan is to create a separate p10bmc-ips machine in bitbake. That will allow us to build a flash image for just IPS. This should simplify a lot of these patches as we can now just install the correct files into the bitbake image and stop keying off of vpd BR.

Reply:
Lets just move this PR over to the IPS branch since it's done and tested.